src/main/java/seedu/address/ui/EntryCard.java

```diff
@@ -24,35 +24,36 @@
      * As a consequence, UI elements' variable names cannot be set to such keywords
      * or an exception will be thrown by JavaFX during runtime.
      *
-     * @see <a href="https://github.com/se-edu/addressbook-level4/issues/336">The issue on AddressBook level 4</a>
+     * @see <a href="https://github.com/se-edu/addressbook-level4/issues/336">The issue on EntryBook level 4</a>
      */
 
-    public final Person person;
+    public final Entry entry;
 
     @FXML
     private HBox cardPane;
     @FXML
-    private Label name;
+    private Label title;
     @FXML
     private Label id;
     @FXML
-    private Label phone;
+    private Label comment;
     @FXML
     private Label address;
     @FXML
-    private Label email;
+    private Label link;
     @FXML
     private FlowPane tags;
 
-    public PersonCard(Person person, int displayedIndex) {
+    public EntryCard(Entry entry, int displayedIndex) {
         super(FXML);
-        this.person = person;
+        this.entry = entry;
         id.setText(displayedIndex + ". ");
-        name.setText(person.getName().fullName);
-        phone.setText(person.getPhone().value);
-        address.setText(person.getAddress().value);
-        email.setText(person.getEmail().value);
-        person.getTags().forEach(tag -> {
+        title.setText(entry.getTitle().fullTitle);
+        comment.setText(entry.getComment().value);
+        address.setText(entry.getAddress().value);
+        address.setManaged(false);
```

Comment:
What does this line do? I think we should add a comment about it here for future reference.

Reply:
Makes the Address label invisible, graphics pipeline will not manage it. And agreed, will do